ml-agents/mlagents/trainers/trainer/rl_trainer.py

```diff
@@ -256,6 +258,19 @@ def _maybe_save_model(self, step_after_process: int) -> None:
         if step_after_process >= self._next_save_step and self.get_step != 0:
             self._checkpoint()
 
+    def _warn_if_group_reward(self, buffer: AgentBuffer) -> None:
+        """
+        Warn if the trainer receives a Group Reward but isn't a multiagent trainer (e.g. POCA).
+        """
+        if not self._has_warned_group_rewards:
+            group_reward = np.sum(buffer[BufferKey.GROUP_REWARD])
```

Comment:
is sum enough? 
```suggestion
            group_reward = np.sum(np.abs(buffer[BufferKey.GROUP_REWARD]))
```

Reply:
If the user only uses group penalties, it will not be caught.

Reply:
Nice catch

Reply:
Replaced with `np.any`, which is faster and doesn't have this issue.